VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
```

Comment:
> Fair point, but you're add one extra method by doing it this way. I guess I prefer the shorter code.

Yeah, the constructor needs to be explicitly defined as private, but making it a standard class removes those pesky extra methods in the public API.

> Also, using `enum` additionally prevents `newInstance()` from working, where-as it will still work if you just use a private constructor.

Which is why I implemented it to throw an `UnsupportedOperationException` upon instantiation 😉

> /pointless argument :laughing:

Agreed.